testFilters/testFilters.php

```diff
@@ -808,6 +809,31 @@ public function getWorkableIntegrationPrePostParams(array $params): array
 		return $params;
 	}
 
+	/**
+	 * Change form fields data before we send it to the external integration.
+	 *
+	 * @param array<string, mixed> $params Array of params.
+	 *
+	 * @return array<string, mixed>
+	 */
+	/**
+	 *
+	 * Change integration ID fields data before we send it to the external integration.
+	 *
+	 * @param string $itemId Item id.
+	 * @param array<mixed> $params Params to alterate.
+	 * @param string $formId Form id.
+	 *
+	 * @return string
+	 */
```

Comment:
```suggestion
	/**
	 * Change integration ID fields data before we send it to the external integration.
	 *
	 * @param string $itemId Item id.
	 * @param array<mixed> $params Params to alterate.
	 * @param string $formId Form id.
	 *
	 * @return string
	 */
```

Double docblock comment

Reply:
tnx